sidekick/test/plugins_test.dart

```diff
@@ -105,29 +101,51 @@ void main() {
               .directory('test/templates/minimal_sidekick_plugin')
               .path;
 
-          final dashProcess = await startDashProcess([
+          await runDashProcess([
             'plugins',
             'install',
             '--source',
             'path',
             pluginPath,
           ]);
-          printOnFailure(await dashProcess.stdoutStream().join('\n'));
-          printOnFailure(await dashProcess.stderrStream().join('\n'));
-          dashProcess.shouldExit(0);
 
-          final pluginProcess = await startDashProcess(
+          await runDashProcess(
             ['minimal-sidekick-plugin'],
           );
-          printOnFailure(await pluginProcess.stdoutStream().join('\n'));
-          printOnFailure(await pluginProcess.stderrStream().join('\n'));
-          pluginProcess.shouldExit(0);
         },
         timeout: const Timeout(Duration(minutes: 5)),
       );
     },
   );
 
+  group('plugins create generates valid plugin code', () {
+    // TODO: use CreatePluginCommand.templates.keys instead (import 'package:sidekick_core/src/commands/plugins/create_plugin_command.dart';)
```

Comment:
doable now

Reply:
In the latest `sidekick_core` (v0.7.1) `CreatePluginCommand.templates` is still private, so `sidekick` can't access the attribute yet